src/foam/core/JSON.js

```diff
@@ -435,6 +435,13 @@ foam.CLASS({
             a[i] = this.objectify(o[i]);
           }
           return a;
+        },
+        Object: function(o) {
+          var ret = {};
+          for ( var key in o ) {
+            if ( o.hasOwnProperty(key) ) ret[key] = this.objectify(o[key]);
```

Comment:
Add a note that a faster version could only create 'ret' when the first objectified object is different than o[key]. This would avoid creating new objects when not necessary.